src/sections/Companies/style.module.scss

```diff
@@ -0,0 +1,19 @@
+.container {
+  background: linear-gradient(180deg, #9d7587 30%, #0a0842 160%);
```

Comment:
I think this shade of pink is slightly off from the section above. See below image. Make sure you're using the variable color values and test out that everything is working correctly in your preview.

![Screenshot 2022-11-09 at 12 22 07 PM](https://user-images.githubusercontent.com/33165426/200933954-5706c581-8513-420e-b7f1-a4cede7044c8.jpg)